### ISSUE TITLE
Default config of `without_timestamps=True` affects whisper transcript quality.

### ISSUE THREAD

**Issue by Artaches** [m-bain/whisperX#932]
WhisperX default is `without_timestamps=True`; while faster-whisper's default is `without_timestamps=False`. This affects transcript quality, whisperX output can have long (5-15s) continuous transcript drops. Attached an example of a small audio clip (~10s, so VADs in WhisperX and Faster-Whisper are off) that has worse transcript output when `without_timestamps=True`.  
[whisperxWithoutTimestepsExample.zip](https://github.com/user-attachments/files/17907624/whisperxWithoutTimestepsExample.zip)

**Comment by pramadikaegamo** [comment]
> WhisperX default is `without_timestamps=True`; while faster-whisper's default is `without_timestamps=False`. This affects transcript quality, whisperX output can have long (5-15s) continuous transcript drops. Attached an example of a small audio clip (~10s, so VADs in WhisperX and Faster-Whisper are off) that has worse transcript output when `without_timestamps=True`. [whisperxWithoutTimestepsExample.zip](https://github.com/user-attachments/files/17907624/whisperxWithoutTimestepsExample.zip)

yes that's right, when without_timestamps=False some of the previously untranscribed ones become transcribed by changing them to false

**Comment by heartInsert** [comment]
where  should  I   change  this  parameter  without_timestamps   to   False  in   whsperx  ?

**Comment by pramadikaegamo** [comment]
> where should I change this parameter without_timestamps to False in whsperx ?

in the load_model function you can add the asr_options parameter, as in the following example:

asr_options = {
"without_timestamps": False,
}

model = whisperx.load_model("large-v3", device="cuda", asr_options=asr_options)

**Comment by heartInsert** [comment]
@pramadikaegamo  thanks  a lot .

**Comment by sulutian** [comment]
> > 我应该在 whsperx 中的哪里将此参数 without_timestamps 更改为 False？
> 
> 在 load_model 函数中您可以添加 asr_options 参数，如下例所示：
> 
> asr_options = { “without_timestamps”：False， }
> 
> 模型 = whisperx.load_model（“large-v3”，设备=“cuda”，asr_options=asr_options）

Can he only use this parameter in python and not on the command line?

**Comment by pramadikaegamo** [comment]
> > > 我应该在 whsperx 中的哪里将此参数 without_timestamps 更改为 False？
> > 
> > 
> > 在 load_model 函数中您可以添加 asr_options 参数，如下例所示：
> > asr_options = { “without_timestamps”：False， }
> > 模型 = whisperx.load_model（“large-v3”，设备=“cuda”，asr_options=asr_options）
> 
> Can he only use this parameter in python and not on the command line?

Yes, of course, that's what I gave, for example using parameters in Python, not in the command line
